container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/reverse_proxy_functions/network_reverse_proxy.go

```diff
@@ -48,6 +53,11 @@ func DisconnectReverseProxyFromNetwork(ctx context.Context, dockerManager *docke
 		return nil
 	}
 
+	_, found := maybeReverseProxyObject.GetEnclaveNetworksIpAddress()[networkId]
```

Comment:
Does there need to be a check for `maybeReverseProxyObject` not being `nil`?

Reply:
Cleaned that up too.